src/CMakeLists.txt

```diff
@@ -29,34 +16,42 @@ set(ALUT_SOURCES
 	alutOutputStream.c
 	alutUtil.c
 	alutVersion.c
-	alutWaveform.c)
-
-set(ALUT_INTERNAL_HEADERS
-	alutInternal.h)
-source_group(InternalHeader FILES ${ALUT_INTERNAL_HEADERS})
-
-set(ALUT_HEADERS
-	../include/AL/alut.h)
-source_group(APIHeader FILES ${ALUT_HEADERS})
+	alutWaveform.c
+	${ALUT_INTERNAL_HEADERS}
+	${ALUT_HEADERS})
 
-add_library(alut ${ALUT_SOURCES} ${ALUT_INTERNAL_HEADERS} ${ALUT_HEADERS})
-set_property(TARGET
+set_target_properties(
 	alut
-	PROPERTY
-	PUBLIC_HEADER
-	${ALUT_HEADERS})
-set_target_properties(alut
-	PROPERTIES
-	VERSION
-	${MAJOR_VERSION}
-	SOVERSION
-	${MAJOR_VERSION})
-target_link_libraries(alut ${OPENAL_LIBRARY})
+	PROPERTIES PUBLIC_HEADER "${ALUT_HEADERS}"
+			   VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_VERSION}
```

Comment:
This regresses a change introduced in 6fdc3652705ac33967062bea27e921969460d4cb.

HOWEVER, the change in *that* commit made it so the Autotools build installed different lib names from CMake.

Your change here shifts it back to them both installing `libalut.so.0.1.0`, which I think we want.

Reply:
It should also install a symlink libalut.so.0